### PR TITLE
Improve maps_generator README.

### PR DESCRIPTION
**Pull request by hallahan**
I tweaked some of the English wording in this doc. My intent is to make this a bit easier to understand to a newcomer that wants to generate some .mwm files themselves.

cc/ @endim8 @biodranik 

Inspired by:
https://github.com/organicmaps/organicmaps/issues/4392

Related issues:
https://github.com/organicmaps/organicmaps/issues/4405
https://github.com/organicmaps/organicmaps/issues/4406

### CODE REVIEW

File: tools/python/maps_generator/README.md [hunk 1]
```diff
@@ -26,35 +23,44 @@ git checkout release-92
 
 The app version can be found in the "About" section in the settings menu of OMaps.
 
-1. [Build and install generator_tool.](https://github.com/organicmaps/organicmaps/blob/master/docs/INSTALL.md#desktop-app)
-2. Change directory:
+2. Build and install the generator_tool.
+
+```sh
+./tools/unix/build_omim.sh -r desktop
```
Comment (biodranik) [inline tools/python/maps_generator/README.md:29]: You don't need the desktop app for the generator.

File: tools/python/maps_generator/README.md [hunk 2]
```diff
@@ -26,35 +23,44 @@ git checkout release-92
...
 
-3. Install dependencies:
+1. Install dependencies:
 
 ```sh
 maps_generator$ pip3 install -r requirements_dev.txt
 ```
 
-4. Make ini file:
+1 Make the ini configuration file:
```
Comment (biodranik) [inline tools/python/maps_generator/README.md:46]: Why 1. ?

Did you preview it? Was the markdown with numbered items rendered correctly?

File: tools/python/maps_generator/README.md [hunk 3]
```diff
@@ -26,35 +23,44 @@ git checkout release-92
...
 ```
 
-5. Edit ini file:
+5. Edit the ini file:
 
 ```sh
 maps_generator$ vim var/etc/map_generator.ini
 ```
 
+Here is a sample ini that will download an OSM PBF file for the Yukon Territories, Canada from [GEOFABRIK](https://www.geofabrik.de/). You can replace the *osm.pbf* and *osm.pbf.md5* with other areas instead. Note that an entire planet file currently takes 40+hrs on a server with 256GB of RAM. Unless you are have a machine this large, it is recommended to use a smaller extract.
```
Comment (biodranik) [inline tools/python/maps_generator/README.md:58]:
```suggestion
Here is a sample ini that will download an OSM PBF file for the Yukon Territories, Canada from [GEOFABRIK](https://www.geofabrik.de/). You can replace the *osm.pbf* and *osm.pbf.md5* with other areas instead. Note that an entire planet file currently takes 40+ hours on a server with 256GB of RAM. Unless you have a machine this large, it is recommended to use a smaller extract.
```

File: tools/python/maps_generator/README.md [hunk 4]
```diff
@@ -63,17 +69,15 @@ MAIN_OUT_PATH: ~/maps_build
...
 BUILD_PATH: ~/omim-build-release
-# The path to the project directory omim.
-OMIM_PATH: ~/omim
+# The path to the organicmaps repo
+OMIM_PATH: ~/code/organicmaps
 
 
 [Generator tool]
-# The path to the omim/data.
+# The path to the application data.
```
Comment (biodranik) [inline tools/python/maps_generator/README.md:79]:
```suggestion
# The path to the data folder in the repository.
```

File: tools/python/maps_generator/README.md [hunk 5]
```diff
@@ -124,25 +140,33 @@ SUBWAY_URL: https://cdn.organicmaps.app/subway.json
...
+cd tools/python
+python -m maps_generator --countries="Canada_Yukon_North, Canada_Yukon_Whitehorse" --skip="Coastline"
+```
 
-## Usage
+## Help
 
 ```sh
-$ cd omim/tools/python
 python$ python3.6 -m maps_generator -h
```
Comment (biodranik) [inline tools/python/maps_generator/README.md:170]:
```suggestion
python$ python -m maps_generator -h
```

File: tools/python/maps_generator/README.md [hunk 6]
```diff
@@ -195,22 +219,19 @@ optional arguments:
   --order ORDER         Mwm generation order.
 ```
 
-If you are not from the maps.me team, then you do not need the option --production when generating maps.
+If you are not from the OrganicMaps team, then you do not need the option --production when generating maps.
```
Comment (biodranik) [inline tools/python/maps_generator/README.md:222]:
```suggestion
If you are not from the Organic Maps team, then you do not need the option --production when generating maps.
```

File: tools/python/maps_generator/README.md [hunk 7]
```diff
@@ -228,26 +249,29 @@ DEBUG: 0
 PLANET_URL: https://download.geofabrik.de/asia/japan-latest.osm.pbf
 PLANET_MD5_URL: https://download.geofabrik.de/asia/japan-latest.osm.pbf.md5
 ...
+
+To build an entire country with coastlines, you need to download the *latest_coasts.geom* and *latest_coasts.rawgeom* files and specify their path in the config.
```
Comment (biodranik) [inline tools/python/maps_generator/README.md:253]: It is only needed in the case of building some regions. If the whole planet is built, then these files will be generated/used directly.

File: tools/python/maps_generator/README.md [hunk 8]
```diff
@@ -228,26 +249,29 @@ DEBUG: 0
...
 
-#### Rebuild stages:
+### Rebuild stages
 
-For example, you changed routing code in omim project and want to regenerate maps.
+For example, you changed routing code in the project and want to regenerate maps.
 You must have previous generation. You may regenerate from stage routing only for two mwms:
 
 ```sh
 python$ python3.6 -m maps_generator -c --from_stage="Routing" --countries="Japan_Kinki Region_Osaka_Osaka, Japan_Chugoku Region_Tottori"
```
Comment (biodranik) [inline tools/python/maps_generator/README.md:269]:
```suggestion
python$ python -m maps_generator -c --from_stage="Routing" --countries="Japan_Kinki Region_Osaka_Osaka, Japan_Chugoku Region_Tottori"
```

Please check if there are other 3.6 hardcoded versions somewhere.

File: tools/python/maps_generator/README.md [hunk 9]
```diff
@@ -272,12 +296,12 @@ PLANET_MD5_URL: https://download.geofabrik.de/russia/central-fed-district-latest
 3. Run
 
 ```sh
-python$ python3.6 -m maps_generator --countries="Russia_Moscow" --skip="Coastline"
+python$ python -m maps_generator --countries="Russia_Moscow" --skip="Coastline"
```
Comment (biodranik) [inline tools/python/maps_generator/README.md:299]:
```suggestion
python$ python -m maps_generator --countries="Japan_Chugoku Region_Tottori" --skip="Coastline"
```

**Review by biodranik — CHANGES_REQUESTED** [review]

File: tools/python/maps_generator/README.md (hunk 2 above)
Comment (hallahan) [reply tools/python/maps_generator/README.md:46]: Whoops, fixing.

File: tools/python/maps_generator/README.md (hunk 7 above)
Comment (hallahan) [reply tools/python/maps_generator/README.md:253]: Are you saying that this part of the config can be left blank in a planet build? For this example, we are doing all of Japan. 

I have found that the generator fails if I include World and WorldCoasts, at least for the Yukon. So aren't the geom files always required?

Comment (biodranik) [reply tools/python/maps_generator/README.md:252]: You don't need to download these files if the whole planet is built. They are generated in the process of building the whole planet.

**Review by biodranik — APPROVED** [review]

**Pull request merged**